### DISCUSSION TITLE
1.7.5 detected as trojan virus

### DISCUSSION THREAD

**Discussion by gaveitatry2** [r57zone/DualShock4-emulator#38] [General]
On October 14, 2021, I downloaded DS4 Emulator 1.7.0 Portable.  I didn't use the program.  I just extracted it, but I was able to successfully extract the contents without getting a trojan virus alert.  And in the past, I have used older versions of DS4 Emulator and never received a trojan virus alert.

But today, I downloaded DS4 Emulator 1.7.5 Portable, and I immediately got a severe trojan virus alert
![ApplicationFrameHost_Oy1Xg2cElz](https://user-images.githubusercontent.com/75463665/143284162-af48bcdb-e86b-4117-b4a4-99dfbe580ecb.png)
I know a lot of people online say things like "don't worry" and "it's just a false alert".  But is there anything you can do to make a DS4 Emulator 1.7.5 Portable zip file without this trojan?  I haven't subscribed to PlayStation Now yet, but I will subscribe this Saturday or Sunday.

**Comment by gaveitatry2** [comment]
**UPDATE:**

1.74 also has the same trojan alert.  1.73 won't even download because Windows 10 detected a virus.
![firefox_mqFe83xydH](https://user-images.githubusercontent.com/75463665/143285617-7075d0f3-7152-4ddd-868d-5a3ae0dbe9e2.png)
1.72 and 1.71 are safe - no warnings.

**Reply by VeganOCR** [reply]
I get 3 alerts on virustotal.com for DS4 Emulator and 1 alert for ViGEmbus today.
![DS4_Virustotal_Alert](https://user-images.githubusercontent.com/101937657/185806595-aed22824-cb43-403a-9c03-600a5d0ab1d0.PNG)
![Vigembus_Virustotal_Alert](https://user-images.githubusercontent.com/101937657/185806596-20e7bc9f-2521-4828-9440-714882b700c6.PNG)

**Reply by r57zone** [reply]
It's not antivirus it's crap

**Comment by r57zone** [comment]
I sent the latest release to Microsoft for research, hopefully they'll fix it. Perhaps this is due to the hiding of the cursor, but I will not delete it. These shitty antiviruses constantly see viruses in something that is not a virus.

**Comment by r57zone** [comment]
![](https://user-images.githubusercontent.com/9499881/143292962-ed037a3c-8991-46c7-9ace-24e6150cac04.png)

**Reply by gaveitatry2** [reply]
That's funny.  LOL.  Thank you for the reply.

**Comment by r57zone** [comment]
AVG antivirus specialists manually checked and removed false positives  ![Screenshot 2](https://user-images.githubusercontent.com/9499881/143478374-d8a0c1e1-a3ae-43f2-ae9e-546daf030ad8.png)

**Reply by e-t-l** [reply]
I did the same thing, submitting 1.7.4 to Kaspersky, and now Kaspersky has stopped flagging DS4Emulator (incl 1.7.5). Might just have to manually submit every time it gets flagged, and eventually all the major AVs will have it sorted out. Idk how responsive Microsoft is about this stuff, though, so if you're using Windows Defender you might be out of luck

**Comment by r57zone** [comment]
Newer versions don't have this problem, so I'm closing it. Garbage antiviruses can be ignored.
